app/src/main/java/com/sduduzog/slimlauncher/adapters/AddAppAdapter.kt

```diff
@@ -28,17 +28,15 @@ class AddAppAdapter(private val listener: OnAppClickedListener) : RecyclerView.A
         return ViewHolder(view)
     }
 
-    fun setItems(apps: List<App>){
-        this.apps = apps
+    fun setItems(apps: List<App>) {
+        this.apps.addAll(apps)
```

Comment:
Just a hunch, I think this will just append to the list, where else the point was to completely replace the contents of the previous list. I haven't built the branch yet to test

Reply:
Good catch, I added a clear before adding.